data/cube/View.js

```diff
@@ -110,8 +115,12 @@ export class View extends HoistBase {
         const newQuery = this.query.clone(overrides);
         if (this.query.equals(newQuery)) return;
 
-        // If anything other than just filter changing, blow away row cache
-        if (!isEqual(keys(overrides), ['filter'])) {
+        this._simpleAggs = newQuery.fields.every(
```

Comment:
a static function taking the query seems  appropriate here isQuerySimple ( or querySupportsStreaming ) ? and above lines 69-71